master_changes.md

```diff
@@ -131,7 +131,10 @@ users)
 ## opam-format
 
 ## opam-core
```

Comment:
Maybe it worth having an entry in `#Internal`, to highlight the better system polling change

Reply:
done